src/main/resources/study.cql

```diff
@@ -1,9 +1,38 @@
 CREATE TABLE IF NOT EXISTS study.study (
-    studyName text PRIMARY KEY,
+    studyName text,
     networkUuid uuid,
     networkId text,
     description text,
     caseFormat text,
     caseUuid uuid,
-    casePrivate boolean
+    casePrivate boolean,
+    userId text,
+    isPrivate boolean,
+    PRIMARY KEY (studyName, userId)
```

Comment:
At least, the primary key should be userId, studyName.

Maybe for now the simplest solution is to do

CREATE TABLE IF NOT EXISTS privateStudy
     PRIMARY KEY (userId, studyName) 

CREATE TABLE IF NOT EXISTS publicStudy
     PRIMARY KEY (userId, studyName) 

I believe these two tables are enough for all our queries.